app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):

```

Comment:
![image](https://user-images.githubusercontent.com/105493988/219084537-b9516421-30b3-4aad-bda5-e9b133fbc261.png)
Do it according to the task description, fix everywhere